### ISSUE TITLE
Consistent way to set CSS for EPUB cover

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/publ-cg#98]
I think we need a consistent way to set CSS for EPUB cover.

In some reading systems cover images are splitting in multiple virtual pages, e.g.:

<img width="1552" alt="Schermata 2020-03-25 alle 15 04 39" src="https://user-images.githubusercontent.com/5065125/77563452-983b7b80-6ec1-11ea-8ad0-667c8ef118e7.png">
<img width="1552" alt="Schermata 2020-03-25 alle 15 04 47" src="https://user-images.githubusercontent.com/5065125/77563465-9bcf0280-6ec1-11ea-9c92-e9608c424ba3.png">

**Comment by llemeurfr** [comment]
I started discussing this with Jiminy Panoz, the author of the Blitz framework. 
Notes:
- this is a general issue for images which must be displayed full-page in a reflowable resource.
- resolving that issue in EPUB files for all kinds of reading apps is reaallly cumbersome (see [this Blitz issue](https://github.com/FriendsOfEpub/Blitz/issues/25) for a proposed solution).
- a good solution would be IMO to propose a fairly simple best practice which works for most reading apps, especially those supporting EPUB 2 (vs EPUB 2); e.g. based on object-fit. And files issues for the reading apps which don't behave correctly when this best practice is applied.

**Comment by gregoriopellegrino** [comment]
What about using a declarative solution based on dpub ARIA (https://www.w3.org/TR/dpub-aria-1.0/)?

In this case `role="doc-cover"`

**Comment by clapierre** [comment]
I like this approach with using `doc-cover` and then shouldn't cause issues where an image was meant to cross two page spreads, although not sure if that is one of the use-cases we are trying not to interfere with.

**Comment by dauwhe** [comment]
I would much prefer that we document best practices rather than asking reading systems to perform special rendering based on markup.

The original images illustrate a massive bug. Images [don't fragment](https://drafts.csswg.org/css-break-3/#breaking-rules), period, no matter what CSS is applied to them. 

> Some content is not fragmentable, for example many types of replaced elements [CSS2] (such as images or video), scrollable elements, or a single line of text content. Such content is considered monolithic: it contains no possible break points. Any forced breaks within such boxes therefore cannot split the box, and must therefore also be ignored by the box’s own fragmentation context.

**Comment by llemeurfr** [comment]
Generally speaking, dpub ARIA is about accessibility only; this is not a generic way to add generic semantic tagging to EPUB. 
More, a descriptive tagging won't help reading apps which rely on webviews.

**Comment by gregoriopellegrino** [comment]
Yeah, I realized now that using dpub ARIA wouldn't be really "webby".
